source/jormungandr/jormungandr/api.py

```diff
@@ -74,7 +74,11 @@ def access_log(response, *args, **kwargs):
         'status': response.status_code,
         'external_request_id': request.headers.get('x-request-id'),
     }
-    logger.info(u'"%(method)s %(path)s?%(query_string)s" %(status)s', d, extra=d)
+    if 'readyness' not in request.path:
+        logger.info(u'"%(method)s %(path)s?%(query_string)s" %(status)s', d, extra=d)
+    else:
+        logging.getLogger(__name__).debug(u'"%(method)s %(path)s?%(query_string)s" %(status)s', d, extra=d)
```

Comment:
is there a reason for not using `logger` here as above ?
```suggestion
        logger.debug(u'"%(method)s %(path)s?%(query_string)s" %(status)s', d, extra=d)
```

Reply:
Will test this this suggestion.

Reply:
This doesn't work as the logger is initialized with level = INFO.